network_insight_sdk_generic_datasources/archive/zip_archiver.py

```diff
@@ -4,7 +4,7 @@
 import os
 import zipfile
 from shutil import copyfile
-from network_insight_sdk_generic_datasources.common.log import py_logger
+from common.log import py_logger
```

Comment:
import should not be changed. Please check your PYTHONPATH. Same goes for all of imports.
REF: https://github.com/vmware/network-insight-sdk-generic-datasources#3-launch-sdk

Reply:
I have docuto make that change because just following the procedure that was in error...

Reply:
export PYTHONPATH="$PYTHONPATH:../network-insight-sdk-generic-datasources"
You mean using above PYTHONPATH you got error.

Reply:
export PYTHONPATH="$PYTHONPATH:../network-insight-sdk-generic-datasources" will work